ajax/libs/jquery-nearest/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "jquery-nearest",
+    "description": "Find the elements in a page that are closest to a particular point or element, based on pixel dimensions",
+    "filename": "jquery.nearest.min.js",
+    "keywords": [
+        "jquery-plugin", 
+        "ecosystem:jquery", 
+        "nearest", "furthest", 
+        "touching", 
+        "pixel", 
+        "distance"
+    ],
+    "author": "Gilmore Davidson",
```

Comment:
Let's follow the upstream https://github.com/gilmoreorless/jquery-nearest/blob/master/package.json#L15-L17